comp/otelcol/otlp/components/logsagentpipeline/agent_core_init.go

```diff
@@ -0,0 +1,52 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021-present Datadog, Inc.
+
+package logsagentpipeline
```

Comment:
you may want to merge this with `agent.go`. it was only in another file in the core agent because of build tags.